06-Customizing AzTS for your org/Extending AzTS/Prerequisites.md

```diff
@@ -42,29 +42,51 @@ Below mentioned appsettings are required for enabling CMET:
 
 > **Note:** **FeatureManagement__PolicyStates** appsettings is required only for Azure custom policy-based control evaluation.
 -----
+### Option2: Using helper script
```

Comment:
Can we call this something better? (Same comment as above)

Reply:
changed to Using Powershell